qa/suites/basic/health-rgw-ssl.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+# DeepSea integration test "suites/basic/health-rgw.sh"
+#
+# This script runs DeepSea stages 0-4 to deploy a Ceph cluster with RGW.  After
+# stage 4 completes, it sends a GET request to the RGW node using curl, and
+# tests that: (a) the response contains the string "anonymous" and (b) the
+# response is legal XML.
```

Comment:
This looks like the same description as for `health-rgw.sh` . . . can it be updated?